templates/definition/meter/senec-home.yaml

```diff
@@ -17,6 +17,12 @@ params:
       - http
       - https
     default: http
+  - name: live_stats
```

Comment:
Could we make this `livestats`? Think we typically don't use underscores. Could also rename this to `version` with v3 and v2 as values? This could be useful for later extensions.